handles.go

```diff
@@ -52,12 +52,12 @@ func (v *HandleList) Track(pointer interface{}) unsafe.Pointer {
 
 	v.Unlock()
 
-	return unsafe.Pointer(&slot)
+	return unsafe.Pointer(uintptr(slot))
```

Comment:
Ok, I see why we have to do this. But this means that the `set` map is now not really keeping the pointers alive. It should then move to `map[int]bool` since we only need it to be set.